lib/Backends/Habana/Habana.cpp

```diff
@@ -1092,6 +1092,7 @@ HabanaBackend::compile(Function *F, const BackendOptions &opts) const {
         std::vector<synTensor> inputs;
         inputs.push_back(tensors[MI->getLHS()].get());
         inputs.push_back(tensors[MI->getRHS()].get());
+        inputs.push_back(nullptr);
```

Comment:
Can you add `/* bias */`? It's not really intuitive to think of a MatMul with a Bias.